app/Http/Controllers/Projects/JobOrderController.php

```diff
@@ -100,6 +101,25 @@ public function edit($id)
     }
 
     /**
+     * Display materials based from the specified resource.
+     *
+     * @param  int  $id
+     *
+     * @return \Illuminate\View\View
+     */
+    public function materials($id)
```

Comment:
@jandyl-casas no need to create new method for the materials, use the MaterialsController